src/features/swaps/SwapScreen.tsx

```diff
@@ -183,11 +182,10 @@ const SwapScreen = () => {
 
     setSolFee(fee)
 
-    const balance = await connection.getBalance(
-      new PublicKey(currentAccount.solanaAddress),
+    setHasInsufficientBalance(
+      fee > solBalance.integerBalance || solBalance.floatBalance < 0.000005,
```

Comment:
Is the || case just in there to default while the fee loads?

Reply:
Yup